protos/bottle/inventory/v1/location.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package bottle.inventory.v1;
+
+message Location {
+  string id = 1;
```

Comment:
I plan to just use the `uuid` we have on locations and ignore the auto increment.

Reply:
Sweet. That's kinda what I was expecting / hoping when we started splitting things off and using string for id.